extensions/wikia/Captcha/CaptchaHooks.class.php

```diff
@@ -58,4 +58,12 @@ public static function APIGetAllowedParams( &$module, &$params ) {
 	public static function APIGetParamDescription( &$module, &$desc ) {
 		return Factory\Module::getInstance()->APIGetParamDescription( $module, $desc );
 	}
+
+	// Temporarily load captcha library on every page until we have on demand loading set up.
+	// Remove with https://wikia-inc.atlassian.net/browse/SOC-288
+	public static function onSkinAfterBottomScripts( $skin, &$text ) {
```

Comment:
Ok, this could sacrifice site performance and generate unnecessary load. Is there a solid reason why to introduce this temporary change (loading captcha always) instead of loading in on demand right away (within this PR)?